includes/class-wc-gateway-stripe.php

```diff
@@ -916,27 +926,32 @@ public function prepare_intent_for_order_pay_page( $order = null ) {
 	}
 
 	/**
-	 * Renders hidden inputs on the "Pay for Order" page in order to let Stripe handle PaymentIntents.
+	 * Renders hidden inputs on the "Pay for Order" page in order to let Stripe handle intents.
```

Comment:
Some older parts of the code say simply intents when they mean (only) PaymentIntents. Could we make it explicit whether this code handles both?